libs/blocks/locui/loc/index.js

```diff
@@ -97,12 +97,23 @@ async function loadProjectSettings(projSettings) {
   }
 }
 
+function sanitize(url) {
+  let newURL = url;
+  const matchHost = /^https?:\/\/[^/?#]+/g;
+  const [hostname] = url.match(matchHost);
+  if (hostname) {
+    const enDash = hostname.replaceAll('–', '--');
+    newURL = url.replace(matchHost, enDash);
+  }
+  return new URL(newURL);
```

Comment:
Wouldn't a lack of a hostname indicate a faulty URL that might yield errors when using `new URL`? Comparing it to the previous logic, this is ok, since this was happening before as well. But from a contained `sanitize` method perspective, it may be counter-intuitive to not check the validity of the URL before returning it.